test/info.py

```diff
@@ -144,6 +145,8 @@
     (   0x1237,     'sam3u2c_ublox_evk_nina_b1_if',            'sam3u2c_bl',        'U-BLOX-EVK-NINA-B1'                    ),
     (   0xC006,     'lpc11u35_vbluno51_if',                     None,               'VNG-VBLUno51'                          ),
     (   0xC005,     'lpc11u35_mtconnect04s_if',                 None,               'MtConnect04S'                          ),
+    (   0xC006,     'lpc11u35_vbluno51_if',                     None,               'VNG-VBLUno51'                          ),    
```

Comment:
This board isn't using the tmpm366. Any reason it needs to be added in this PR? Also did you want to include it in the release table above?

Reply:
Removed the duplicate configuration